python/pyspark/sql/tests/pandas/helper/helper_pandas_transform_with_state.py

```diff
@@ -1663,10 +1663,17 @@ def close(self) -> None:
 
 # A stateful processor that contains composite python type inside Value, List and Map state variable
 class PandasStatefulProcessorCompositeType(StatefulProcessor):
+    from typing import NamedTuple
```

Comment:
Can we move this to the top of the test module? Is there a specific reason that you want it in the class definition?